docs/8-veba-troubleshooting.md

```diff
@@ -230,3 +243,35 @@ Disconnecting from vCenter Server ...
 
 2020/03/11 22:15:15 Duration: 6.085448 seconds
 ```
+
+An alternative way to troubleshoot OpenFaaS logs is to use faas-cli
+
+This faas-cli command will show all available functions in the appliance. ```--tls-no-verify``` bypasses SSL certificate validation
+
+```
+faas-cli list --tls-no-verify
```

Comment:
This assumes `faas-cli` is set up correctly (e.g. logged in and pointing to VEBA). You might want to explain this explicitly as we do in the function examples.

Reply:
faas-cli setup is already covered in the existing getting started guide, and will also be referenced in part IV, deploying the first sample function. Do we need to cover it again in the troubleshooting guide?

Reply:
No, I think we're good. Thx